src/submission/models.py

```diff
@@ -906,15 +906,30 @@ def metrics(self):
     def has_galley(self):
         return self.galley_set.all().exists()
 
+    @cache(600)
+    def publication_detail_settings(self):
+        display_date_accepted = self.journal.get_setting(
+            group_name='article',
+            setting_name='display_date_accepted',
+        )
+        display_date_submitted = self.journal.get_setting(
+            group_name='article',
+            setting_name='display_date_submitted',
+        )
+        return display_date_submitted, display_date_accepted
+
```

Comment:
This is an instance method, so the cache won't behave across article instances (A view with 20 articles would still miss the cache 20 times). Consider either adding it to the `journal_settings` we store on the `request` object OR turn this method into a classmethod/staticmethod.